modules/web/src/app/settings/admin/interface/template.html

```diff
@@ -82,8 +82,22 @@
         </div>
         <mat-checkbox [(ngModel)]="settings.enableDashboard"
                       (change)="onSettingsChange()"
-                      id="km-enable-kubernetes-dashboard-setting"></mat-checkbox>
-        <km-spinner-with-confirmation [isSaved]="isEqual(settings.enableDashboard, apiSettings.enableDashboard)"></km-spinner-with-confirmation>
+                      [disabled]="!isKubernetesDashboardFeatureGatesEnabled()"
+                      id="km-enable-kubernetes-dashboard-setting">
+          <ng-container *ngIf="!isKubernetesDashboardFeatureGatesEnabled()">
+
+            <mat-hint *ngIf="!isOIDCKubeCfgEndpointEnabled">This feature is disabled. Visit the
```

Comment:
I think we should also show some hint when `isOpenIDAuthPluginEnabled` is `false` or can it be the same hint in both cases?